### DISCUSSION TITLE
Ubuntu image 6.6.8 on X96 MAX Plus (S905X3)

### DISCUSSION THREAD

**Discussion by JuneJaJa** [devmfc/debian-on-amlogic#24] [General]
I tested your latest image on X96 Max Plus by installing gnome desktop. Everything looks pretty good. I have some questions to ask you.

1) I can't change the cursor color. (It will always be green)
2) Want to increase the size of the swap file. How can I do it? (Currently 7xx MB)

Thank you for doing good things for us.

**Comment by devmfc** [comment]
Good to hear. And you're welcome. 

Unfortunately, I don't use these boxes for desktop usage because I think they are waaay too slow for that. So I also don't know much about Linux desktops and can't answer you questions. 
But, it is standard Ubuntu (except for the cursor maybe) so google should just know the answer ;-)

**Reply by JuneJaJa** [reply]
Thank you for your answer.

As far as testing goes, it's not too bad. Except for various video work where the resolution should not exceed 720p 30fps.

Now it's another model of my hk1box (S905X4). Looking forward to your work on the next release. Hope to get good news for this release with Linux desktop work.

thank you

**Reply by devmfc** [reply]
The MVP player can play video with hardware decoding. But I never got it to work with one of the browsers.

You realize that S905X4 is very experimental, so it can not be as good as S905X3 with my images (No Linux HDMI support) . But good luck, HK1 should be the same as the Vontar X4, so there is a config for it. Let me now if it works, than I'll add it to the supported boxes list.

Have fun!

**Reply by JuneJaJa** [reply]
> You realize that S905X4 is very experimental, so it can not be as good as S905X3 with my images (No Linux HDMI support) . But good luck, HK1 should be the same as the Vontar X4, so there is a config for it. Let me now if it works, than I'll add it to the supported boxes list.
> 
Yes, I tested on HK1 RBOX X4 (RAM4 ROM128), system booted up fine, VFD displayed correctly, netword can be used. 

Linux desktop is not yet available. which you noted, I know well
